python/datasources/covid_tracking_project.py

```diff
@@ -14,5 +17,112 @@ def get_id():
     def get_table_name():
         return 'covid_tracking_project'
 
+    @staticmethod
+    def get_standard_columns():
+        """Returns a dict containing conversions from Covid Tracking Project's
+           race categories to their standardized values."""
+        return {
+            'aian': col_std.Race.AIAN.value,
+            'asian': col_std.Race.ASIAN.value,
+            'black': col_std.Race.BLACK.value,
+            'nhpi': col_std.Race.NHPI.value,
+            'white': col_std.Race.WHITE.value,
+            'multiracial': col_std.Race.MULTI.value,
+            'other': col_std.Race.OTHER.value,
```

Comment:
I think maybe we should split OTHER into two (not sure on the exact naming):
1. The same "Other" that matches up with ACS "Other"
2. A different "Other" that can't be joined with ACS

If a dataset uses the exact same race categories as ACS then we can join the "other" group, otherwise we can't. Even if we mostly just use the non-joinable one, I think being explicit would be better

Reply:
Sure, fine to not block on this. If there's a possibility you don't get to this before the end of the fellowship, do you mind filing a github issue in case you don't finish?

I think the "defines_other" doesn't necessarily give all the information we need to know? Even if "other" is defined, it still needs to be defined in the exact same way as the ACS "other" category, which may not always be the case?

Reply:
Created #309